lib/src/converter/args_to_protubuf_converter.dart

```diff
@@ -10,8 +10,6 @@ class ArgsToProtobufConverter {
     Map<Uuid, List<Uuid>> servicesWithCharacteristicsToDiscover,
     Duration connectionTimeout,
   ) {
-    assert(id != null);
```

Comment:
I removed this since we do not do this consistenly for other methods in this class

Reply:
Sounds fine to me. If we decide we need such asserts, it would be better to scatter them over the whole library consistently.